src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -49,7 +53,16 @@ public double getTargetAngle(double Y, double X) { // Gets the target angle from
     return angle;
   }
 
-  
+  public double interpolateAngle(double key) {
+    return angleMap.get(key);
+  }
+
+  public double estimateDistance() {
+    /*Gets angle offset by adding mount angle and how far off the apriltag is from crosshair and converts to radians*/
+    double angleToGoal = (ManipulatorConstants.CAMERA_MOUNT_ANGLE_DEGREES + y) * (3.14159 / 180.0);
+    //calculate distance
+    return (ManipulatorConstants.GOAL_HEIGHT - ManipulatorConstants.CAMERA_HEIGHT) / Math.tan(angleToGoal);
```

Comment:
if goal height and camera height are both 40, (like in the `Constants.java`) this will be `0`.